client/src/main.ts

```diff
@@ -99,8 +99,10 @@ function getServerOptions(
   config: Config
 ): lc.ServerOptions {
   let args = ['lsp'];
-  if (config.debug.showParsedTokensAsWarnings) {
-    args.push(' --parsed-tokens-as-warnings');
+  let debug_tokens = config.debug.showCollectedTokensAsWarnings;
+  if (debug_tokens != 'off') {
```

Comment:
There's no functional difference in this case, but it's a best practice to always use triple equals for equality checks, (except in a few rare cases) because javascript double equals behaves in unexpected ways.

```suggestion
  if (debug_tokens !== 'off') {
```

Reply:
Cool, good to know!